Client/src/Pages/Monitors/CurrentStats.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect, useState } from "react";
+import Host from "../../Components/Host";
+import HostStatus from "../../Components/HostStatus";
+import HostActions from "../../Components/HostActions";
+import BarChart from "../../Components/Charts/BarChart/BarChart";
+import Button from "../../Components/Button";
+import ServerStatus from "../../Components/Charts/Servers/ServerStatus";
+import CurrentMonitors from "../../Components/CurrentMonitors";
+import MockData from "../../Mock/sample_data.json";
+
+const hardCodedMonitors = [
+  {
+    host: Host(
+      "Discord",
+      100,
+      "var(--env-var-color-17)",
+      "https://www.google.com"
+    ),
+    status: HostStatus(
+      "var(--env-var-color-20)",
+      "Up",
+      "var(--env-var-color-17)"
+    ),
+    team: <BarChart />,
+    actions: HostActions(),
+  },
+  {
+    host: Host(
+      "Google",
+      99.9,
+      "var(--env-var-color-17)",
+      "https://www.google.com"
+    ),
+    status: HostStatus(
+      "var(--env-var-color-20)",
+      "Up",
+      "var(--env-var-color-17)"
+    ),
+    team: <BarChart />,
+    actions: HostActions(),
+  },
+  {
+    host: Host(
+      "NBC",
+      98.1,
+      "var(--env-var-color-18)",
+      "https://www.google.com"
+    ),
+    status: HostStatus(
+      "var(--env-var-color-20)",
+      "Up",
+      "var(--env-var-color-17)"
+    ),
+    team: <BarChart />,
+    actions: HostActions(),
+  },
+  {
+    host: Host(
+      "Google",
+      95.1,
+      "var(--env-var-color-19)",
+      "https://www.google.com"
+    ),
+    status: HostStatus(
+      "var(--env-var-color-21)",
+      "Down",
+      "var(--env-var-color-19)"
+    ),
+    team: <BarChart />,
+    actions: HostActions(),
+  },
+  {
+    host: Host(
+      "NBC",
+      99.9,
+      "var(--env-var-color-17)",
+      "https://www.google.com"
+    ),
+    status: HostStatus(
+      "var(--env-var-color-15)",
+      "Cannot resolve",
+      "var(--env-var-color-22)"
+    ),
+    team: <BarChart />,
+    actions: HostActions(),
+  },
+];
```

Comment:
I don't think we need this hardcoded data anymore since we have at least mock data now right?

Reply:
Yes. I'll remove it

Reply:
Done